### PR TITLE
BOTMETA - plugins/filters

### PR DESCRIPTION
**Pull request by gundalow**
##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
```
core.py
__init__.py
ipaddr.py
json_query.py
k8s.py
mathstuff.py
network.py
urlsplit.py
```

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -906,11 +906,19 @@ files:
 ###############################
 # plugins/filter
   $plugins/filter/:
+    support: community
+  $plugins/filter/__init__.py:
```
Comment (bcoca) [inline .github/BOTMETA.yml:910]: doesn't this need to be just the dir?

Comment (gundalow) [reply .github/BOTMETA.yml:910]: `$plugins/filter/` is `community, unless specified otherwise

**Comment by maxamillion** [comment]
rebuild_merge

**Pull request merged**